ocw/lib/openstack.py

```diff
@@ -23,14 +24,17 @@ def __new__(cls, vault_namespace: str):
     def client(self) -> None:
         if self.__client is None:
             self.__client = openstack.connect(
+                debug=DEBUG,
```

Comment:
I am not sure that it is good idea to keep it constantly on DEBUG , it can spam logs a lot . Instead we defining some common log level for whole app here https://github.com/SUSE/pcw/blob/master/templates/pcw.ini#L11 what if you will use this one ? Or at least define dedicated variable in `pcw.ini` for it . I want to have ability to mute it w/o tweaking the code

Reply:
It can be muted w/o tweaking the code because `debug` is only `True` if set here:
https://github.com/SUSE/pcw/blob/master/webui/settings.py#L30